Updates the language on the serverless init cloud run page to push customers to use the sidecar (#27101)
DataDog/documentation: content/en/serverless/guide/gcr_serverless_init.md

## Overview

> <div class="alert alert-info">Note, if you are running multiple containers per Google Cloud Run application the recommended approach is to use the Datadog sidecar: <a href="/serverless/google_cloud_run">Instrument Google Cloud Run</a>.</div>

Reviewer: ```suggestion
<div class="alert alert-info">If you are running multiple containers per Google Cloud Run application, Datadog recommends using the Datadog sidecar: <a href="/serverless/google_cloud_run">Instrument Google Cloud Run</a>.</div>
```
